### ISSUE TITLE
Add contrast ratio text layer for failed layers

### ISSUE THREAD

**Issue by romannurik** [romannurik/Sketch-Contrast#1]
This is in the code now, but was commented out because it doesn't look very good. Should bring it back. (+1 from Paul)
